test/bdd/steps/lib/local-blockchain.mjs

```diff
@@ -42,9 +42,12 @@ const testParametersStorageParams = {
  *
  * @param {String} [options.logger] - Logger instance with debug, trace, info and error methods.
  */
+
+let startBlockchainProcess;
```

Comment:
Why? We're not changing this variable anywhere from what I see, why not just `const` as it was?